doc/api/net.markdown

```diff
@@ -723,8 +723,8 @@ Returns true if input is a version 6 IP address, otherwise returns false.
 [`pause()`]: #net_socket_pause
 [`resume()`]: #net_socket_resume
 [`server.getConnections()`]: #net_server_getconnections_callback
-[`server.listen(port, \[host\], \[backlog\], \[callback\])`]: #net_server_listen_port_hostname_backlog_callback
-[`socket.connect(options\[, connectListener\])`]: #net_socket_connect_options_connectlistener
+[`server.listen(port, host, backlog, callback)`]: #net_server_listen_port_hostname_backlog_callback
```

Comment:
Oh, and I just notice that I misread your suggestion completely, please disregard me :)